keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ServerGroup.kt

```diff
@@ -45,6 +45,7 @@ data class ServerGroup(
   val health: Health = Health(),
   val scaling: Scaling = Scaling(),
   val tags: Map<String, String> = emptyMap(),
+  val onlyEnabledServerGroup: Boolean = true,
```

Comment:
Suggestion to make it even clearer:
```suggestion
  val isOnlyEnabledServerGroup: Boolean = true,
```

Reply:
I'm gonna keep this as is to match titus, and because the suggested name is pretty close. It doesn't seem worth adding the "is"